.stylelintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "stylelint-config-standard"
```

Comment:
@gyopiazza I created a netz Stylelint ruleset recently, you can see it used in NUTR-LOPRO: https://github.com/netzstrategen/nutr-lopro/blob/master/.stylelintrc.json